File: admin/lib/dto/inventory/Medication_import_request.dart
class MedicationImportRequest {
  final String name;
  final String description;
  final String unit;
  final int price;
  final int quantity;
  final int categoryId;

  MedicationImportRequest({
    required this.name,
    required this.description,
    required this.unit,
    required this.price,
    required this.quantity,
    required this.categoryId,
  });

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'description': description,
      'unit': unit,
      'price': price,
      'quantity': quantity,
      'categoryId': categoryId,
    };
  }
}

File: be/src/main/java/com/example/petclinic/dto/request/inventory/MedicationImportRequest.java
package com.example.petclinic.dto.request.inventory;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.experimental.FieldDefaults;

@Data
@AllArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class MedicationImportRequest {
    String name;
    String description;
    String unit;
    int price;
    int quantity;
    int categoryId;
}
